app/Console/UpdateUniqueOrganizers.php

```diff
@@ -129,14 +129,15 @@ private function updateOrganizer(Uuid $organizerUuid, Url $organizerUrl): void
         // 2. The organizer uuid is not present
         // 3. The organizer uuid is present with another value
 
-        $existingOrganizerUrl = $this->connection->createQueryBuilder()
+        $existingOrganizerUrls = $this->connection->createQueryBuilder()
             ->select('unique_col')
             ->from('organizer_unique_websites')
             ->where('uuid_col = :uuid')
             ->setParameter('uuid', $organizerUuid->toString())
             ->execute()
             ->fetchAll(PDO::FETCH_COLUMN);
 
+        $existingOrganizerUrl = \count($existingOrganizerUrls) === 1 ? $existingOrganizerUrls[0] : null;
```

Comment:
Nit: You don't need the `\` in `\count()` here. And most of the existing code doesn't use FQN for functions in the root namespace.

Reply:
Indeed! Must be something added by the new version of PHPStorm.